src/test/java/com/cottonfactory/products/controllers/ShoeControllerTest.java

```diff
@@ -0,0 +1,55 @@
+package com.cottonfactory.products.controllers;
+
+import com.cottonfactory.products.entities.ShoeProduct;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.math.BigDecimal;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
```

Comment:
Does it  need `@AutoConfigureTestDatabase` to validate entry to H2 during setup ?

Reply:
testcase is working fine without @AutoConfigureTestDatabase so not making any change